dbt_project.yml

```diff
@@ -1,6 +1,7 @@
 name: "dbt_fullstory"
 version: "0.1.0"
 config-version: 2
+require-dbt-version: [">=1.2.0", "<2.0.0"]
```

Comment:
`v1.2.0` was chosen as a minimum version because of the introduction of [cross-database macros](https://docs.getdbt.com/reference/dbt-jinja-functions/cross-database-macros), which I believe is the latest dbt-core feature we rely on. Unfortunately, I was unable to build/test `dbt-core v1.2.0` with warehouse adapters on my machine because of build toolchain incompatibilities. We should either find a way to test the old version or bump to `v1.6.0`. My concern is that this may be too restrictive, and those running an older version won't be able to install the package.

Reply:
What was failing?

Reply:
There is some transitive build dependency, `pyarrow`, that can't be built. I don't have time to look into it, but it does look like a common problem!